spyder/plugins/editor/widgets/codeeditor/lsp_mixin.py

```diff
@@ -1187,6 +1187,14 @@ def _apply_document_edits(self, edits):
             merged_text = merged_text.replace("\n", self.get_line_separator())
             cursor = self.textCursor()
 
+            # Save breakpoints here to restore them after applying inserting
+            # merged_text
```

Comment:
```suggestion
            # Save breakpoints here to restore them after inserting merged_text
```

Sorry @jsbautista, there was a small error in my previous suggestion.